xlml/apis/metric_config.py

```diff
@@ -37,6 +37,10 @@ class AggregationStrategy(enum.Enum):
   MEDIAN = enum.auto()
 
 
+class MetricPath(enum.Enum):
```

Comment:
nit: let's make this more general (like `SshEnvVars`) and put any future common variables we add to the ssh environment here.